config.yml

```diff
@@ -20,6 +20,9 @@ decentralandApi:
 
 logLevel: 'DEBUG' # Set LOG_LEVEL env variable to overwrite this value
 
+infura:
```

Comment:
Maybe a more general name than "infura" for this???

Reply:
Any suggestion? Why we can change for web3 rpc, I feel like nothing is as clear as infura right now

Reply:
maybe rpcConnection